mis_builder/views/mis_builder.xml

```diff
@@ -0,0 +1,179 @@
+                <script type="text/javascript" src="/mis_builder/static/src/js/mis_builder.js"></script>
+            </xpath>
+        </template>
+
+        <record model="ir.ui.view" id="mis_report_view_tree">
+            <field name="name">mis.report.view.tree</field>
+            <field name="model">mis.report</field>
+            <field name="arch" type="xml">
+                <tree string="MIS Reports">
+                    <field name="name"/>
+                    <field name="description"/>
+                </tree>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="mis_report_view_form">
+            <field name="name">mis.report.view.form</field>
+            <field name="model">mis.report</field>
+            <field name="arch" type="xml">
+                <form string="MIS Report" version="7.0">
+                <sheet>
+                    <group col="2">
+                        <field name="name"/>
+                        <field name="description"/>
+                        <field name="query_ids">
+                            <tree string="Queries" editable="bottom">
+                                <field name="name"/>
+                                <field name="model_id"/>
+                                <field name="field_ids" domain="[('model_id', '=', model_id)]" widget="many2many_tags"
+                                                        on_change="onchange_field_ids(field_ids, context)"/>
+                                <field name="field_names"/>
+                                <field name="date_field" domain="[('model_id', '=', model_id), ('ttype', 'in', ('date', 'datetime'))]"/>
+                                <field name="domain"/>
+                            </tree>
+                        </field>
+                        <field name="kpi_ids">
+                            <tree string="KPI's" editable="bottom">
+                                <field name="sequence" widget="handle"/>
+                                <field name="description" on_change="onchange_description(description, name, context)"/>
+                                <field name="name" on_change="onchange_name(name, context)"/>
+                                <field name="expression"/>
+                                <field name="type" on_change="onchange_type(type, context)"/>
+                                <field name="dp" attrs="{'invisible': [('type', '=', 'str')]}"/>
+                                <field name="divider" attrs="{'invisible': [('type', '=', 'str')]}"/>
+                                <field name="suffix"/>
+                                <field name="compare_method" attrs="{'invisible': [('type', '=', 'str')]}"/>
+                                <field name="default_css_style"/>
+                                <field name="css_style"/>
+                            </tree>
+                        </field>
+                    </group>
+                </sheet>
```

Comment:
Add help to explain how to use. (example : https://github.com/odoo/odoo/blob/8.0/openerp/addons/base/ir/ir_sequence_view.xml#L26)